docs/pipelines/yaml-schema.md

```diff
@@ -875,6 +876,83 @@ steps:
 ::: moniker-end
 
 ## Resources
+Any external service that is consumed as part of your pipeline is a resource. An example of a resource can be another CI/CD pipeline that produces artifacts (say Azure pipelines, Jenkins etc.), code repositories (GitHub, Azure Repos, Git), container image registries (ACR, Docker hub etc.).
+
+Resources in YAML represent sources of types pipelines, repositories and containers.
+
+#### Schema
```

Comment:
Incorrect heading, should be `###`

Reply:
I'd also probably call this "General schema" or something, to differentiate it from the explicit schemas you'll show below.

Reply:
Done